build-docker.gradle

```diff
@@ -52,7 +52,7 @@ task createDockerfileDev(type: Dockerfile) {
     if (project.hasProperty('dockerPrepareEnvironment') && dockerPrepareEnvironment?.trim()) {
         runCommand(dockerPrepareEnvironment)
     }
-    if (project.hasProperty('sealightsSession')) {
+    if (project.hasProperty('sealightsSession') && !project.getProperty('sealightsSession').isEmpty()) {
```

Comment:
Lets check for emptiness in groovy style:
sealightsSession?.trim()

Reply:
Fixed